mesheryctl/internal/cli/root/system/model.go

```diff
@@ -56,10 +56,10 @@ var listModelCmd = &cobra.Command{
 	Long:  "list name of all registered models",
 	Example: `
 // View list of models
-mesheryctl exp model list
+mesheryctl system model list
```

Comment:
@singh1203 Could you please clarify why we're moving the model command into the system when it's still in the experimental stage?